src/super_gradients/modules/detection_modules.py

```diff
@@ -20,10 +27,361 @@ def __init__(self, in_channels: Union[List[int], int], **kwargs):
     @property
     @abstractmethod
     def out_channels(self) -> Union[List[int], int]:
+        raise NotImplementedError()
+
+
+class NStageBackbone(BaseDetectionModule):
```

Comment:
Whats the use of this class in this PR..?

Reply:
Gone, changed merged in #490